pyproject.toml

```diff
@@ -113,3 +114,14 @@ exclude_lines = [
     "raise NotImplementedError",
     "if __name__ == .__main__.:",
 ]
+
+[tool.mypy]
+ignore_missing_imports = true
```

Comment:
It seems like highlighting missing imports would be useful. Why ignore missing imports?

Reply:
Makes sense, thanks for the explanation.